src/states/game.js

```diff
@@ -61,6 +80,17 @@ Game.prototype =
         this.__sprites.blackHole.body.mass = this.__parameters.blackHole_mass;
         
         this.__non_player_objects.push(this.__sprites.blackHole);
+        
+        this.maxScore = 0;
+        this.scoreText = this.add.text(this.world.centerX, 40,
+                                       this.maxScore.toFixed(0),
+                                       {
+                                           font: '62px Roboto Mono',
+                                           fill: "#ffffff",
+                                           align: "center"
+                                       });
+        this.scoreText.anchor.set(0.5);
+        this.offset_x = 0;
     },
 
     update : function()
```

Comment:
Getting a bit bloated in the update function, might be a good idea to split this out into different funcs